### ISSUE TITLE
Telescope create branch filter

### ISSUE THREAD

**Issue by polarmutex** [ThePrimeagen/git-worktree.nvim#31]
When you use telescope to create a worktree filter out branches that already have worktree that should cause an error if you did create them

**Comment by matu3ba** [comment]
Can you rephrase this into a readable sentence?

**Comment by polarmutex** [comment]
basically when creating a worktree using treesitter, we should filter out branch names that already have a worktree associated with them (since git worktree will not let you create a second tree from a branch)
